Update rosettanet-connector-reference.adoc (#357)
mulesoft/docs-connectors: modules/ROOT/pages/rosettanet/rosettanet-connector-reference.adoc

| Service Endpoint Path a| String |  Service endpoint path, appended to base path on HTTP Listener.|  | x
| Global HTTP Request a| String |  Reference to the global HTTP Request definition for partner service endpoint used when sending both action and signal messages. |  | x
| Request Endpoint Path a| String |  Partner endpoint path, appended to base path on HTTP Request. |  | x
| Keystore path a| String |  Keystore containing trusted partner certificates and our own private signing key and certificate (JKS format).|  |
| Keystore access password a| String |  Keystore access password. |  |
| Signing key password a| String |  Private signing key password (required if signing either action or signal messages). |  |
| Override message signing a| Enumeration, one of:

** ALWAYS
** NEVER |  Force message signing on or off (when set, overrides signing configured by PIP definition). |  |
| Global usage code a| Enumeration, one of:

** Test
** Production |  Choose whether operating in Test or Production mode, or if both allowed.|  Production|

> | Object store reference a| Object Store |  Object store configuration reference. If not set the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retrys. If set, the referenced bean must be an object store configuration to use instead. |  |

Reviewer: ```suggestion
| Object store reference a| Object Store |  Object store configuration reference. If not set, the connector always uses the default persistent object store to retain sent messages waiting for acknowledgments or retries. If set, the referenced bean must be an object store configuration. |  |
```

Reviewer 2: Fixed.

==== Parameters
[%header,cols="20s,20a,35a,20a,5a"]
|===
| Name | Type | Description | Default Value | Required
| Configuration | String | The name of the configuration to use. | | x
| Initiating action a| <<ReplyAttributes>> |  |  |
| Content a| Binary |  |  `#[payload]` |
| Streaming Strategy a| * <<repeatable-in-memory-stream>>
* <<repeatable-file-store-stream>>
* non-repeatable-stream |  Configure to use repeatable streams. |  |
| Target Variable a| String |  The name of a variable to store the operation's output. |  |
| Target Value a| String |  An expression to evaluate against the operation's output and stores the expression outcome in the target variable. |  `#[payload]` |
|===

==== Output
[%autowidth.spread]
|===
|Type |Binary
| Attributes Type a| <<MessageAttributes>>
|===

=== For Configurations
* <<config>>

==== Throws
* ROSETTA:WRITE
* ROSETTA:CONFIGURATION
* ROSETTA:PARSE
* ROSETTA:UNKNOWN

== Sources

[[actionSource]]
=== Action Source
`<rosetta:action-source>`

==== Parameters
[%header,cols="20s,20a,35a,20a,5a"]
|===
| Name | Type | Description | Default Value | Required
| Configuration | String | The name of the configuration to use. | | x

> | Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster. |  |

Reviewer: ```suggestion
| Primary Node Only a| Boolean |  Whether this source should be executed only on the primary node when running in a cluster. |  |
```

| Streaming Strategy a| * <<repeatable-in-memory-stream>>
* <<repeatable-file-store-stream>>
* non-repeatable-stream |  Configure to use repeatable streams. |  |
| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message. |  |
|===

==== Output
[%autowidth.spread]
|===
|Type |Binary
| Attributes Type a| <<MessageAttributes>>
|===

=== For Configurations
* <<config>>

[[completionSource]]
=== Completion Source
`<rosetta:completion-source>`

==== Parameters
[%header,cols="20s,20a,35a,20a,5a"]
|===
| Name | Type | Description | Default Value | Required
| Configuration | String | The name of the configuration to use. | | x

> | Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster. |  |

Reviewer: ```suggestion
| Primary Node Only a| Boolean |  Whether this source should be executed only on the primary node when running in a cluster. |  |
```
